designer/server/src/routes/account/auth.js

```diff
@@ -26,24 +24,15 @@ export default [
       // Add to authentication cookie for session validation
       cookieAuth.set({ sessionId: credentials.user.id })
 
-      /**
-       * Drop user sessions when scopes are missing but let users sign back in
-       * @todo Keep users signed in but restrict route access by scope
-       */
-      if (!credentials.scope.includes(scopes.SCOPE_WRITE)) {
-        await dropUserSession(request)
-
-        // Redirect users to a help message
-        yar.flash(sessionNames.userAuthFailed, true)
```

Comment:
Note to other reviewers, we will pick this back up in the next PR.

Reply:
Resolved in https://github.com/DEFRA/forms-designer/pull/385/commits/8e801b47a3b3fbe31e064af0d74283eaf3f3b1a7